common/defaults-gcp.yaml

```diff
@@ -1,3 +1,230 @@
-# TODO: add GCP defaults
-region: us-east4
+cloud_provider: gcp
+cluster_name: dmtr-cluster
+region: us-west-2
```

Comment:
This can't be correct for GCP.

Reply:
Good catch my config.yaml has those defined and it overrides it